website/vercel.json

```diff
@@ -38,6 +38,16 @@
     }
   ],
   "redirects": [
+    {
+      "source": "/reference/macro-properties",
```

Comment:
this redirect can be removed. We shouldn't be deleting the macro properties page.